src/server.ts

```diff
@@ -1,13 +1,48 @@
+import { AxiosResponse } from "axios";
 import express, { Express } from "express";
  
-const app = express();
+const expressApp = express();
 const port = 8000;
+const axios = require('axios');
+
+function getTest(expressRequest: express.Request, expressResponse: express.Response): void {
+    expressResponse.send("Hello wolrd !");
+}
+
+function getRovers(expressRequest: express.Request, expressResponse: express.Response): void {
+    axios.get('https://api.nasa.gov/mars-photos/api/v1/rovers?api_key=IqA8fYkIhjXf62lkAuWM3WqexG4waX0mbovOUeII')
+        .then(function (axiosResponse: AxiosResponse) {
+        let roverNamesList: string[] = [];
+            for (const rover of axiosResponse["data"]["rovers"]) {
+                roverNamesList.push(rover["name"]);
+            }
+        let roverNamesString: string = roverNamesList.join(", ")
```

Comment:
These two `let`s are good candidates to be `const`s

A good rule of thumb is that if you ever have to write a `let`, you can probably think of a way to write it as a `const` instead, and your code will be cleaner as a result :+1: